Source/WebKit/Shared/Notifications/NotificationManagerMessageHandler.h

```diff
@@ -45,13 +46,13 @@ class NotificationManagerMessageHandler : public IPC::MessageReceiver {
 
     virtual void showNotification(IPC::Connection&, const WebCore::NotificationData&, RefPtr<WebCore::NotificationResources>&&, CompletionHandler<void()>&&) = 0;
     virtual void cancelNotification(WebCore::SecurityOriginData&&, const WTF::UUID& notificationID) = 0;
-    virtual void clearNotifications(const Vector<WTF::UUID>& notificationIDs) = 0;
```

Comment:
Why do we remove this? Is this because it's unused? If so, can we do the removal in a separate PR?